backend/layers/business/business.py

```diff
@@ -356,8 +355,15 @@ def get_dataset_artifact_download_data(
 
         file_name = artifact_url.path[1:]
         file_type = artifact.type
-        file_size = self.s3_provider.get_file_size(artifact.uri)
-        presigned_url = self.s3_provider.generate_presigned_url(artifact.uri)
+        try:
```

Comment:
I'm not sure we should do this. If these fail, it's likely an AWS issue and we should probably escalate the error.

Reply:
yeah I made this change to pass a test, but honestly thinking about it, I should change the test instead